src/praisonai-agents/praisonaiagents/permissions/manager.py

```
@@ -351,7 +351,31 @@ def _check_shell_command(
         """
         original_target = prefix + command
         try:
-            from .command_parser import parse_command
+            from .command_parser import parse_command, has_unresolvable_expansion
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)